servers/ros2/free_fleet_server/src/Server.cpp

```diff
@@ -143,6 +144,10 @@ bool Server::is_ready()
 
 void Server::start()
 {
+  fleet_to_rmf_transform = math::convert(server_config.transformation);
+  fleet_to_rmf_yaw = fleet_to_rmf_transform.eulerAngles(0, 1, 2);
```

Comment:
this causes a build error for me. replacing it with 
```
fleet_to_rmf_yaw = fleet_to_rmf_transform.eulerAngles(0, 1, 2)(0);
```
allows it to build. fleet_to_rmf_yaw is type `double`, but `.eulerAngles` returns a `Vector3f`

Reply:
gotcha, my bad